app/views/passengers/index.haml

```diff
@@ -43,7 +43,7 @@
           %td= passenger.email
           %td= passenger.phone
           %td.text-center{data: {sort: passenger.permanent?.to_s}}= yes_no_image passenger.permanent?
-          %td.text-center{data: {sort: sortable_date(passenger.doctors_note)}}=passenger.expiration_display
+          %td.text-center{data: {sort: sortable_date(passenger.doctors_note)}}=passenger.rides_expire.try(:strftime, '%m/%d/%Y')
```

Comment:
Do you want to make this change to the "archived" view too?  How about the printout?

Reply:
It was a conscious decision not to make this change to the archived view. Since these passengers aren't actively getting rides, I couldn't find a reason why it might be useful, as opposed to knowing when someone's doctor's note expired, which might be.

As for the printout, I plumb forgot.